pace-util/pace/util/quantity.py

```diff
@@ -392,12 +399,19 @@ def storage(self):
             )
         return self._storage
 
-    def _initialize_storage(self, data, origin, gt4py_backend: str):
+    def _initialize_storage(self, data, origin, gt4py_backend: str, mask: Tuple):
+        spatial_dims = [
+            i for j in [constants.X_DIMS, constants.Y_DIMS, constants.Z_DIMS] for i in j
+        ]
+        extra_dims = [i for i in self._dims if i not in spatial_dims]
+        if len(extra_dims) > 0 or not self._dims:
+            mask = None
```

Comment:
Instead of passing `dims` silently through an object attribute, can you put this if statement where `mask` is computed so the passed value for `mask` will be None in this case, and then remove the attribute `self._dims`? This should make it clearer when reading the code that computes the mask that we only compute/use the mask when there are no non-spatial dimensions.